app/models/katello/concerns/host_managed_extensions.rb

```diff
@@ -32,6 +33,15 @@ def rhsm_organization_label
         self.organization.label
       end
 
+      def info_with_katello
+        info = info_without_katello
+        info['parameters']['kt_env'] = self.lifecycle_environment.try(:label) #deprecated
+        info['parameters']['kt_cv'] = self.content_view.try(:label) #deprecated
```

Comment:
I don't know there is a way to document this?  Maybe we should start a katello.org page or wiki page for scheduled deprecations?